examples/java-springboot/src/main/java/com/spruceid/didkitexample/util/DIDKitOptions.java

```diff
@@ -14,4 +19,25 @@ public class DIDKitOptions {
     private String verificationMethod;
     private String challenge;
     private String domain;
+
+    // This will be the "system time" for when something is processed
+    // not the time the VP is created.
+    private String created;
+
+    public DIDKitOptions(
+        String proofPurpose,
+        String verificationMethod,
+        @NonNull Optional<String> challenge,
+        String domain,
+        @NonNull Optional<Instant> created
+    ) {
```

Comment:
I used the Optional class in my last Java project and found that they don't quite solve the problem as well as in other languages like Rust since the Optional object can itself be null if you are not careful.